docs/model/introduction-to-model-architecture.md

```diff
@@ -1,21 +1,21 @@
-# Basics About Model Architecture
+# Introduction to Model Architecture
 
-In this article you will learn about [entities](#entity), [facades](#facade), [repositories](#repository) and their mutual relations.
+In this article you will learn about model, its dependencies, [entities](#entity), [facades](#facade), [repositories](#repository) and their mutual relations.
 
-## Basics about the model
+## Definition of a Model
+Definition of model is adopted form DDD definition of a model. Model is a system of abstractions that describes selected aspect of a domain.
 
-Model architecture of Shopsys Framework is inspired by Domain Driven Design (DDD). Let us define some terms first:
-- **Domain** in DDD is a sphere of knowledge or activity we build application logic around. The domain of Shopsys Framework is e-commerce.
-- **Domain model** is a system of abstractions that describes selected aspects of the domain.
-- **Domain logic** or **business logic** is the higher level rules for how objects of the domain model interact with one another.
+Domain is a sphere of knowledge or activity we build application logic around. The domain of Shopsys Framework is e-commerce.
+
+Each domain has its logic which is the higher level rules for how objects of the domain model interact with one another.
```

Comment:
logic is the higher level rules ? maybe some commas could help with this sentence

Reply:
If you see some specific mistake, please point that out, because in my opinion, this sentence makes sense